### ISSUE TITLE
[Webinar] 16.04 ESM

### ISSUE THREAD

**Issue by pmahnke** [canonical-web-and-design/ubuntu.com#8786]
Create **[Webinar] 16.04 ESM** based on the [brief](https://docs.google.com/spreadsheets/d/19TsEce5LHt9_3lTzGWqOrk4WkpVU7OhDVVjVAGd7Yco/edit)

**Comment by Khutch25** [comment]
> Create **[Webinar] 16.04 ESM** based on the [brief](https://docs.google.com/spreadsheets/d/19TsEce5LHt9_3lTzGWqOrk4WkpVU7OhDVVjVAGd7Yco/edit)

This is the correct brief document: https://docs.google.com/spreadsheets/d/1Bc76x005Tdj056BQ4EHSf4ab92wavCJNjxe7VHKtUOg/edit?usp=sharing

**Comment by Khutch25** [comment]
Okay, so [updated the brief to the new discourse brief template](https://docs.google.com/spreadsheets/d/1rkfh1bf72N90hrXJ12sXBV4_YLkI2E1vamTx1w6MvTw/edit?usp=sharing) @pmahnke @SirSamTumless but ran into some problems when trying to actually create the engage pages themselves.

1. I received this error, saying that only two links were allowed for a new user:
![Screenshot 2020-11-30 at 22 20 07](https://user-images.githubusercontent.com/42180941/100673317-29b99200-335b-11eb-8a81-aaf1a79f9606.png)

2. It wouldn't let me add any pages to the Engage / index topic on discourse in point 4 of the steps at the top of the [Engage Pages tab](https://docs.google.com/spreadsheets/d/1rkfh1bf72N90hrXJ12sXBV4_YLkI2E1vamTx1w6MvTw/edit#gid=0l) '4. Now you need to add the page to the index topic on discourse.'

**Comment by pmahnke** [comment]
@Khutch25 I thought you were going to do the first few paired with a web person?

Clearly you do not have the access rights yet.

**Comment by pmahnke** [comment]
You didn't put it in the right category or add the metadata to it.

I did it again here: https://ubuntu.com/engage/16-04-ESM-webinar

I will say it looks odd with both a whitepaper and webinar.

**Comment by Khutch25** [comment]
> @Khutch25 I thought you were going to do the first few paired with a web person?
> 
> Clearly you do not have the access rights yet.

**Comment by Khutch25** [comment]
> You didn't put it in the right category or add the metadata to it.
> 
> I did it again here: https://ubuntu.com/engage/16-04-ESM-webinar
> 
> I will say it looks odd with both a whitepaper and webinar.

I assume I couldn't add to the right category because I didn't have access to that category. I searched for Engage, takeovers, index, etc. and none appeared for me, and therefore the metadata pieces didn't appear for me.

Would love to go over one with you or someone in the team so I can get access and understand the process :)

Also, there's no whitepaper for this one. Sorry, it looks like when I copied and pasted an old form id for something else in the new brief template. I just removed the data in Rows 29 and 30 in the [new brief](https://docs.google.com/spreadsheets/d/1rkfh1bf72N90hrXJ12sXBV4_YLkI2E1vamTx1w6MvTw/edit#gid=1011867606) but the [old brief](https://docs.google.com/spreadsheets/d/1Bc76x005Tdj056BQ4EHSf4ab92wavCJNjxe7VHKtUOg/edit#gid=0) is good to reference in case I messed up.
